imports/plugins/core/core/server/Reaction/core.js

```diff
@@ -96,6 +96,11 @@ export default {
         functionsByType[type].push(...packageInfo.functionsByType[type]);
       });
     }
+    if (packageInfo.paymentMethods) {
+      for (const paymentMethod of packageInfo.paymentMethods) {
+        paymentMethods.add(paymentMethod);
```

Comment:
For some of the future PRs related to this, it might be necessary to have the name of the plugin that added the method, and just helpful in general. You can do this here:

```js
paymentMethods.add({ ...paymentMethod, pluginName: packageInfo.name  });
```